3.7/graphs-satellite-graphs-details.md

```diff
@@ -0,0 +1,159 @@
+    ~graphModule._drop("normalGraph", true);
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock satelliteGraphExplain3_1_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+Let us analyze a query involving a traversal:
+
+{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+    @startDocuBlockInline satelliteGraphExplain3_2_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{satelliteGraphExplain3_2_cluster}
+    ~var graphModule = require("@arangodb/general-graph");
+    ~var satelliteGraphModule = require("@arangodb/satellite-graph");
+    ~graphModule._create("normalGraph", [ graphModule._relation("edges", "vertices", "vertices") ], [], {});
+    ~satelliteGraphModule._create("satelliteGraph", [ satelliteGraphModule._relation("satEdges", "satVertices", "satVertices") ], [], {});
+    ~db._create("collection", {numberOfShards: 8});
+    db._explain(`FOR doc in collection FOR v,e,p IN OUTBOUND "vertices/start" GRAPH "normalGraph" RETURN [doc,v,e,p]`, {}, {colors: false});
+    ~db._drop("collection");
+    ~satelliteGraphModule._drop("satelliteGraph", true);
+    ~graphModule._drop("normalGraph", true);
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock satelliteGraphExplain3_2_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+You can see that the `TraversalNode` is executed on a Coordinator, and only
+the `EnumerateCollectionNode` is executed on DB-Server. This will happen for
+each of the 8 shards in `collection`.
+
+Let us now have a look at the same query using a SatelliteGraph:
+
+{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+    @startDocuBlockInline satelliteGraphExplain3_3_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{satelliteGraphExplain3_3_cluster}
+    ~var graphModule = require("@arangodb/general-graph");
+    ~var satelliteGraphModule = require("@arangodb/satellite-graph");
+    ~graphModule._create("normalGraph", [ graphModule._relation("edges", "vertices", "vertices") ], [], {});
+    ~satelliteGraphModule._create("satelliteGraph", [ satelliteGraphModule._relation("satEdges", "satVertices", "satVertices") ], [], {});
+    ~db._create("collection", {numberOfShards: 8});
+    db._explain(`FOR doc in collection FOR v,e,p IN OUTBOUND "vertices/start" GRAPH "satelliteGraph" RETURN [doc,v,e,p]`, {}, {colors: false});
+    ~db._drop("collection");
+    ~satelliteGraphModule._drop("satelliteGraph", true);
+    ~graphModule._drop("normalGraph", true);
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock satelliteGraphExplain3_3_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+Note that now the `TraversalNode` is executed on each DB-Server, leading to a
+great reduction in required network communication, and hence potential gains
+in query performance.
```

Comment:
Very good!!
I like this chapter, this gives a lot of details for the curious ones!